src/progress-steps/styled-components.js

```diff
@@ -132,17 +132,12 @@ export const StyledContentTail = styled<StylePropsT>(
       position: 'absolute',
       [$theme.direction === 'rtl' ? 'right' : 'left']: '7px',
       top: 0,
-      height: '100%',
-      paddingBottom: 0,
+      height: `calc(100% + ${$theme.sizing.scale500})`,
```

Comment:
Could you add a comment on why `$theme.sizing.scale500` is used here, like where this specific value usage comes from?

Reply:
There seems to be a gap in prod right now as well, I can fix it though.

As for the value, that was the height of the :after pseudo component that made up the original tail.

![image](https://user-images.githubusercontent.com/2348903/72872758-f752f900-3ca2-11ea-858e-f3cb6146e8fa.png)

Reply:
Hi, ok I fixed it to remove the gap. Used the calculations from the icon itself so that the spacing is more accurate now.

I also refactored the NumberContentTail to also get rid of the :after pseudo element in the same way, and adjusted the values to get rid of the gap.